src/datreant/core/util.py

```diff
@@ -1,19 +1,19 @@
 import os
 
 
-def makedirs(path):
+def makedirs(path, mode=511, exists_ok=False):
```

Comment:
No the default is correct [doc](https://docs.python.org/3.6/library/os.html#os.makedirs). But I did get the name wrong.